internal/apicodec/codec_v2_test.go

```diff
@@ -110,62 +114,63 @@ func (suite *testCodecV2Suite) TestNewCodecV2() {
 	re := suite.Require()
 	testCases := []struct {
 		mode           Mode
-		spaceID        uint32
+		keyspaceMeta   keyspacepb.KeyspaceMeta
```

Comment:
How about keeping the spaceID untouched, and constructing the keyspaceMeta via the spaceID.